rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -689,6 +689,337 @@
       ]
     }
   },
+  {
+    "kind": "connector",
+    "data": {
+      "id": "activemq",
+      "name": "ActiveMQ",
+      "description": "ActiveMQ Message Broker",
+      "icon": "fa-puzzle-piece",
+      "configuredProperties": {},
+      "properties": {
+        "brokerUrl": {
+          "kind" : "property",
+          "displayName" : "Broker URL",
+          "group" : "common",
+          "label" : "common",
+          "required" : true,
+          "componentProperty": true,
+          "type" : "string",
+          "javaType" : "java.lang.String",
+          "deprecated": false,
+          "secret": false,
+          "description" : "Broker URL"
+        },
+        "username" : {
+          "kind": "property",
+          "displayName": "User Name",
+          "group": "security",
+          "label": "common,security",
+          "required": false,
+          "componentProperty": true,
+          "type": "string",
+          "javaType": "java.lang.String",
+          "deprecated": false,
+          "secret": false,
+          "description": "Authorization credential user name"
+        },
+        "password" : {
+          "kind" : "property",
+          "displayName" : "Password",
+          "group": "security",
+          "label": "common,security",
+          "required": false,
+          "componentProperty": true,
+          "type": "string",
+          "javaType": "java.lang.String",
+          "deprecated": false,
+          "secret": true,
+          "description" : "Authorization credential password"
+        }
+      },
+      "actions": [
+        {
+          "id": "io.syndesis:activemq-publish-connector:latest",
+          "name": "Publish Messages",
+          "description": "Publish one-way Text messages on an ActiveMQ Queue or Topic.",
+          "tags": [],
+          "pattern": "To",
+          "actionType": "connector",
+          "descriptor": {
+            "camelConnectorGAV": "io.syndesis:activemq-publish-connector:@syndesis-connectors.version@",
+            "camelConnectorPrefix": "activemq-publish",
+            "inputDataShape": {
+              "type": "*"
+            },
+            "outputDataShape": {
+              "kind": "none"
+            },
+            "propertyDefinitionSteps": [{
+              "name": "Select the JMS Destination",
+              "description": "Specify ActiveMQ Connection Factory properties, including Queue or Topic Name",
+              "properties": {
+                "destinationName": {
```

Comment:
//sidenote Created #528, we could add dynamic metadata support for this

Reply:
It's non trivial. JMS API doesn't support listing destination names. So the alternative is to use JMX/Jolokia to do that, which is dependent on the underlying provider. 
Not to mention security considerations in accessing the JMX interface. 
We can leave the issue as a placeholder for now.

Reply:
Also, ActiveMQ supports creating destinations dynamically, so it might not exist before the user specifies it in the connector.